web_src/js/utils/dom.js

```diff
@@ -258,16 +259,25 @@ export function isElemVisible(element) {
   return Boolean(element.offsetWidth || element.offsetHeight || element.getClientRects().length);
 }
 
+export function getComboMarkdownEditor(el) {
+  return el?._giteaComboMarkdownEditor;
+}
+
```

Comment:
Move this somewhere else, `dom.js` is for general DOM-related stuff and this is too specific to the markdown editor.